src/app/modules/balance-projection/service/projection-service/projection.service.spec.ts

```diff
@@ -13,31 +13,16 @@ describe('ProjectionService', () => {
     expect(service).toBeTruthy();
   });
 
-  describe('caluclateIncomeExpenseChart', () => {
-    describe('with valid data', () => {
-      xit('should create chart data', () => {
-        chart = service.caluclateIncomeExpenseChart(900, 300, 4);
-        expect(chart).toBeTruthy();
-      });
-    });
-    describe('with invalid data', () => {
-      xit('should not create chart data', () => {
-        chart = service.caluclateIncomeExpenseChart(900, 300, 0);
-        expect(chart).toBeFalsy();
-      });
-    });
-  });
-
   describe('calculateBalanceChart', () => {
     describe('with valid data', () => {
       xit('should create balance chart', () => {
```

Comment:
Did we mean to leave this as `xit` instead of `it`?

Reply:
I removed the xit's so that all of our tests ran. Would you rather that they be pending? I can change them back if you would like them to be xit's instead of it's.

Reply:
The tests have been updated from "`xit`" to "`it`" now that the methods they are testing have been implemented.

Reply:
Gotcha. Please push up the tests using `it` and I'll approve.

Reply:
The tests in this snippet were removed since they are testing a method that no longer exists or is needed.